src/stories/containers/Finances/components/FinacesTable/FinancesTable.tsx

```diff
@@ -29,17 +30,19 @@ const FinancesTable: React.FC<Props> = ({ className, breakdownTable, metrics, pe
   const showQuarterly = !isMobile && period === 'Quarterly';
   const showMonthly = desk1440 && period === 'Monthly';
   const arrayMetrics = new Array<number>(iteration).fill(0);
-  const showFooter = true;
-  // Show color for others depending if odd or even
-  const isPair = orderData[`${tables[tables.length - 1]}`]?.length % 2 === 0;
+
+  // Show color for others depending if odd or pair
+  const isPair = showFooterAndCorrectNumber[`${tables[tables.length - 1]}`]?.length % 2 === 0;
```

Comment:
please, note here that it should be "isEven", in English we say "odd or even", not "odd or pair"

Reply:
you are right, done